src/native/libs/System.Security.Cryptography.Native/openssl.c

```diff
@@ -1304,6 +1375,9 @@ static int32_t EnsureOpenSsl10Initialized()
     // Ensure that the error message table is loaded.
     ERR_load_crypto_strings();
 
+    // In OpenSSL 1.0.2-, CRYPTO_EX_INDEX_X509 is 10.
+    g_x509_ocsp_index = CRYPTO_get_ex_new_index(10, 0, NULL, NULL, ExDataDup, ExDataFree);
```

Comment:
> CRYPTO_get_ex_new_index() returns a new index or -1 on failure.

Should we assert that we didn't get `-1` here? What if we do in the wild?

Reply:
There's an assert back at the unified caller that the value isn't -1, which happens to both checks that it got assigned ever and that we didn't get a failure here.